MonsterTrainModdingAPI/Utilities/AssetBundleLoadingInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MonsterTrainModdingAPI
```

Comment:
Wrong namespace; if it's in utilities it should be in the Utilities namespace. That said, it's... not really a utility. Maybe make it an inner class of AssetBundleUtils or something? Not sure what the best approach is.